RoddyCore/src/de/dkfz/roddy/config/loader/ProcessingToolReader.groovy

```diff
@@ -234,60 +221,68 @@ class ProcessingToolReader {
      * @return
      */
     ToolEntry.ToolParameter parseToolParameter(NodeChild child, String toolID) {
-        String type = readAttribute(child, "type")
-        if (type == "file") { //Load a file
+        String type = readAttribute(child, 'type')
+        if (type == 'file') { // Load a file
             return parseFile(child, toolID)
-        } else if (type == "tuple") {
+        } else if (type == 'tuple') {
             return parseTuple(child, toolID)
-        } else if (type == "filegroup") {
+        } else if (type == 'filegroup') {
             return parseFileGroup(child, toolID)
         } else if (type == CVALUE_TYPE_STRING) {
-            ToolStringParameter.ParameterSetbyOptions setby = Enum.valueOf(ToolStringParameter.ParameterSetbyOptions.class, extractAttributeText(child, "setby", ToolStringParameter.ParameterSetbyOptions.callingCode.name()))
+            ToolStringParameter.ParameterSetbyOptions setby =
+                    Enum.valueOf(ToolStringParameter.ParameterSetbyOptions.class,
+                            extractAttributeText(child, 'setby',
+                                    ToolStringParameter.ParameterSetbyOptions.callingCode.name()))
 
-            String pName = readAttribute(child, "scriptparameter")
+            String pName = readAttribute(child, 'scriptparameter')
             ToolStringParameter tsp
             if (setby == ToolStringParameter.ParameterSetbyOptions.callingCode) {
                 tsp = new ToolStringParameter(pName)
             } else {
-                tsp = new ToolStringParameter(pName, extractAttributeText(child, "cValueID"))
+                tsp = new ToolStringParameter(pName, extractAttributeText(child, 'cValueID'))
             }
 
             return tsp
         } else {
-            addLoadErr("The type attribute of a parameter was invalid (${type}) for tool ${toolID}\n" + ConfigurationFactory.ERROR_PRINTOUT_XML_LINEPREFIX + RoddyConversionHelperMethods.toFormattedXML(child, "\n" + ConfigurationFactory.ERROR_PRINTOUT_XML_LINEPREFIX))
+            addLoadErr("The type attribute of a parameter was invalid (${type}) for tool ${toolID}\n" +
+                    ConfigurationFactory.ERROR_PRINTOUT_XML_LINEPREFIX +
+                    RoddyConversionHelperMethods.toFormattedXML(child, "\n" +
+                            ConfigurationFactory.ERROR_PRINTOUT_XML_LINEPREFIX))
```

Comment:
You could place the last ) on the new line, which will help Idea to format the document in a better way.